reboot-core/src/main/kotlin/nl/thanus/reboot/ReBoot.kt

```diff
@@ -13,24 +18,33 @@ import java.nio.file.Paths
 
 private val logger = KotlinLogging.logger { }
 
-fun main(args: Array<String>) {
-    val location = args.firstOrNull()
-    location?.let {
-        logger.info { "ReBooting $it" }
-        parseAndReBoot(it)
+class ReBoot : CliktCommand() {
+    val location by argument(help = "Location of project that will be ReBooting")
+    val excludedRefactorings: List<Refactoring> by option("--excluded", "-e", help = """
```

Comment:
Properties can be private
```suggestion
    private val location by argument(help = "Location of project that will be ReBooting")
    private val excludedRefactorings: List<Refactoring> by option("--excluded", "-e", help = """
```